### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by root-expert — APPROVED** [review]
LGTM

**Comment by bastelfreak** [comment]
let's wait with this until https://github.com/voxpupuli/modulesync_config/pull/731 is ready

File: Gemfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ group :release do
 end
 
 gem 'puppet_forge', '>= 2.2.9'
-gem 'metadata_json_deps', '~> 0.2.0'
+gem 'metadata_json_deps', '>= 0.2.0', '< 2'
```
Comment (ekohl) [inline Gemfile:26]: Feels a bit odd to do in a release PR. Not that important since it doesn't affect moduleroot, but still.

Comment (bastelfreak) [reply Gemfile:26]: moved it to https://github.com/voxpupuli/modulesync_config/pull/746

**Review by ekohl — APPROVED** [review]

**Pull request merged**